client.go

```diff
@@ -0,0 +1,204 @@
+package gev
+
+import (
+	"errors"
+	"github.com/Allenxuxu/gev/connection"
+	"github.com/Allenxuxu/gev/eventloop"
+	"github.com/Allenxuxu/gev/log"
+	"github.com/Allenxuxu/toolkit/sync"
+	"github.com/Allenxuxu/toolkit/sync/atomic"
+	"github.com/RussellLuo/timingwheel"
+	"github.com/libp2p/go-reuseport"
+	"golang.org/x/sys/unix"
+	"net"
+	"runtime"
+	"syscall"
+	"time"
+)
+
+type Connector struct {
+	workLoops   []*eventloop.EventLoop
+	opts        *Options
```

Comment:
I think client needs an `options` of his own

Reply:
dunno, should i ? options particularly close same

Reply:
I think it's different from serverOptions